src/app/components/Text/index.tsx

```diff
@@ -11,7 +11,6 @@ type TextProps = {
   maxwidth?: string;
```

Comment:
is better to be maxWidth

Reply:
camel-case naming will throw warning in devtool console, add todo comments